test/integration/targets/docker-registry/tasks/tests/docker_login.yml

```diff
@@ -0,0 +1,102 @@
+---
+- name: Log in with wrong password (check mode)
+  docker_login:
```

Comment:
`check_mode` keyword isn't used, `check_mode` isn't enabled.

Reply:
Thanks, apparently forgot it here.